projects/plugins/jetpack/sal/class.json-api-post-base.php

```diff
@@ -504,30 +834,45 @@ protected function get_avatar_url( $email, $avatar_size = 96 ) {
 	}
 
 	/**
- 	 * Get extra post permalink suggestions
- 	 * @return array	array of permalink suggestions: 'permalink_URL', 'suggested_slug'
- 	 */
+	 * Return extra post permalink suggestions in an array including the 'permalink_URL' and the 'suggested_slug'.
+	 *
+	 * @param string $title The current post title.
+	 *
+	 * @return array
+	 */
 	public function get_permalink_suggestions( $title ) {
 		$suggestions = array();
 		list( $suggestions['permalink_URL'], $suggestions['suggested_slug'] ) = get_sample_permalink( $this->post->ID, $title );
 		return $suggestions;
 	}
 
+	/**
+	 * Returns an object with formatted taxonomy information such as slug and meta information.
+	 *
+	 * Otherwise, returns an error if the edit or display permissions aren't correct.
+	 *
+	 * @param WP_Term $taxonomy The current taxonomy.
+	 * @param string  $taxonomy_type The current taxonomy type, for example 'category'.
+	 * @param string  $context The current context, for example 'edit' or 'display'.
+	 *
+	 * @return object
+	 */
 	private function format_taxonomy( $taxonomy, $taxonomy_type, $context ) {
-		// Permissions
+		// Permissions.
 		switch ( $context ) {
-		case 'edit' :
-			$tax = get_taxonomy( $taxonomy_type );
-			if ( !current_user_can( $tax->cap->edit_terms ) )
-				return new WP_Error( 'unauthorized', 'User cannot edit taxonomy', 403 );
-			break;
-		case 'display' :
-			if ( -1 == get_option( 'blog_public' ) && ! current_user_can( 'read' ) ) {
-				return new WP_Error( 'unauthorized', 'User cannot view taxonomy', 403 );
-			}
-			break;
-		default :
-			return new WP_Error( 'invalid_context', 'Invalid API CONTEXT', 400 );
+			case 'edit':
+				$tax = get_taxonomy( $taxonomy_type );
+				if ( ! current_user_can( $tax->cap->edit_terms ) ) {
+					return new WP_Error( 'unauthorized', 'User cannot edit taxonomy', 403 );
+				}
+				break;
+			case 'display':
+				if ( -1 === get_option( 'blog_public' ) && ! current_user_can( 'read' ) ) {
```

Comment:
I would keep the loose comparison here, I see it used as a string in some places.

```suggestion
				if ( -1 == get_option( 'blog_public' ) && ! current_user_can( 'read' ) ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
```

Reply:
Thanks!